src/scripts/agora.py

```diff
@@ -0,0 +1,164 @@
+################################################################################
+# agora.py
+# 
+# A CLI providing convenient access to the Agora methods repository REST API.
+#
+# author: Bradt
+# contact: dsde-engineering@broadinstitute.org
+# 2015
+# 
+# NOTES:
+# This is a very rough initial implementation. Needs a lot of work.
+#
+# Currently just supports method push. Ulitmately we will want method fetch, and
+#   task-configuration push/fetch.
+#
+# Must find a way to programatically get an Oath token from the openAM server.
+#   Currently, assumes you just grabbed the token and are passing it in as an
+#   argument. Really hackish.
+#
+# Gets default parameters from a .agoraconfig file, assumed to be in the same
+#   directory. Overrides some of them from the command line. I'm not really
+#   happy with the argument parsing/validation here. I would love to get input
+#   from a stronger python programmer. In particular, can we set up different args
+#   args to support multiple 'modes', like FETCH / PUSH.
+# 
+################################################################################
+
+
+from argparse import ArgumentParser
+import os, sys, tempfile, subprocess
+import csv
+import httplib
+import urllib
+import json
+
+ownerConfig = "owner="
+namespaceConfig = "namespace="
+urlConfig = "agora_url="
+
+def fail(message):
+    print message
+    sys.exit(1)
+
+
+# Read defualt values out of the configuration file. In particular, owner.
+# TODO - is there a smarter way to do this?
```

Comment:
You could do this with ConfigParser: https://docs.python.org/2/library/configparser.html